app/scripts/controllers/transactions/lib/util.test.js

```diff
@@ -231,6 +263,22 @@ describe('txUtils', function () {
         );
       });
 
+      it('should error when maxPriorityFeePerGas is not a hex string', function () {
```

Comment:
If you manage to extract the validation part into a function, I would propose an additional test for that function (positive & negative). The tests added here are nice to see as well and should stay.